Resources/Prototypes/XenoArch/artifact_triggers.yml

```
@@ -117,6 +117,49 @@
       effects:
       - !type:ActivateArtifact
 
+- type: artifactTrigger
+  id: TriggerMedical
+  targetDepth: 2
+  triggerHint: artifact-trigger-hint-bioreactive
+  components:
+  - type: reactive
+  groups:
+    Acidic: [ Touch ]
+  reactions:
+  - reagents: [ Dylovene, Diphenhydramine, Arithrazine, Bicaridine, Dermaline, Dexalin, DexalinPlus ] # adding all medical chems seems. Excessive, and also long. Ive just added the main ones for now
+    methods: [ Touch ]
+    effects:
+    - !type:ActivateArtifact
```

I think this has the wrong indentation level? Also, you keep using "reactive" instead of "Reactive" (this is case sensitive).

```suggestion
  components:
  - type: Reactive
    groups:
      Acidic: [ Touch ]
    reactions:
    - reagents: [ Dylovene, Diphenhydramine, Arithrazine, Bicaridine, Dermaline, Dexalin, DexalinPlus ] # adding all medical chems seems. Excessive, and also long. Ive just added the main ones for now
      methods: [ Touch ]
      effects:
      - !type:ActivateArtifact
```

---

Thanks for the review! Pretty sure i've fixed all the indentation and capitalisation issues now hopefully, appreciate it!